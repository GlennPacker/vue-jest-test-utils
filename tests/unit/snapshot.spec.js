import { mount } from '@vue/test-utils';
import HelloWorld from '@/components/HelloWorld.vue';

describe('snapshot examples -', () => {
    it('snapshot basic test', () => {
        const wrapper = mount(HelloWorld);

        expect(wrapper.html()).toMatchSnapshot();
    });
});
