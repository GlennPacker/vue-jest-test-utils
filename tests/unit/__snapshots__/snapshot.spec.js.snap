// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot examples - snapshot basic test 1`] = `
<div class="container">
  <h1>Hello </h1>
  <div>hi </div>
</div>
`;
